src/components/ExpiryNotification/TruncatedAddress.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+export default function TruncatedAddress({ address = '' }) {
+  const showFullAddress = false
+  const addressTruncated = address.substr(0, 14)
+
+  return (
+    <span title={address} style={{ fontWeight: 500 }}>
```

Comment:
Definitely don't want any javascript styles if possible, please change to a styled-component

Reply:
Replaced w. styled-component